hooks/src/utils/card.ts

```
@@ -0,0 +1,50 @@
+import { CARD_BRAND } from "../constants/cardBrand";
+
+type CardBrandType = keyof typeof CARD_BRAND;
+
+export const getCardBrand = (value: string): CardBrandType => {
+  for (const brand in CARD_BRAND) {
+    const condition = CARD_BRAND[brand as CardBrandType].condition;
+    if (condition && condition(value)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)